interpreter/exec/v128.ml

```diff
@@ -233,6 +235,14 @@ struct
   let sub = binop FXX.sub
   let mul = binop FXX.mul
   let div = binop FXX.div
+  let fma x y z =
+    let rec fma_iter xs ys zs : FXX.t list =
+      match xs, ys, zs with
+      | x::xss, y::yss, z::zss -> (FXX.fma x y z) :: (fma_iter xss yss zss)
+      | [], [], [] -> []
+      | _ -> assert false
+    in of_lanes (fma_iter (to_lanes x) (to_lanes y) (to_lanes z))
```

Comment:
```suggestion
  let fma x y z =
    of_lanes (Lib.List.map3 FXX.fma (to_lanes x) (to_lanes y) (to_lanes z))
```
where `util/lib/List` has
```
  let map3 f xs ys zs =
    match xs, ys, zs with
    | [], [], [] -> []
    | x::xs', y::ys', z::zs' -> f x y z :: map3 f xs' ys' zs'
    | _ -> assert false
```

Reply:
Ooo i did not see this lib, neat!

Reply:
Hm, i don't see this https://github.com/WebAssembly/spec/blob/main/interpreter/util/lib.ml#L60

maybe it's in another proposal or another branch? I'll add it in.

Reply:
It's not there yet, but since it's such a generic function, mirroring map2 (which is in the stdlib), I thought it best to factor it out.